src/frontend/src/app/startpage/startpage.component.scss

```
@@ -0,0 +1,116 @@
+.header {
+  display: flex;
+  justify-content: center;
+  overflow-y: hidden;
+  height: auto;
+}
+.iconcontainer {
+  display: flex;
+  justify-content: center;
+  margin-top: 100px;
+}
+.center {
+  display: flex;
+  justify-content: center;
+  overflow-y: hidden;
+  height: auto;
+}
+.share {
+  background-color: white;
+  color: black;
+  transform: scale(2);
+}
+
+.MuiBox-root {
+  align-items: center;
+}
+
+.roundbutton {
+  border-radius: 20px;
+  background: transparent;
+  padding: 6px 12px;
```

```suggestion
  padding: 12px;
```

To center text in button